### PR TITLE
Leaderboard stub

### PR DESCRIPTION
**Pull request by wolfd**
Adds the stub for submitting real scores. Only score possible is 1 now

### CODE REVIEW

File: app/src/main/java/website/bloop/app/dialogs/FlagCapturedDialogFragment.java [hunk 1]
```diff
@@ -1,7 +1,9 @@
 package website.bloop.app.dialogs;
```
Comment (segerphilip) [inline app/src/main/java/website/bloop/app/dialogs/FlagCapturedDialogFragment.java:1]: This doesn't look right?

**Review by segerphilip — APPROVED** [review]
Besides the weird <span> stuff, looks good? 🛡 💃

File: app/src/main/java/website/bloop/app/dialogs/FlagCapturedDialogFragment.java (hunk 1 above)
Comment (segerphilip) [reply app/src/main/java/website/bloop/app/dialogs/FlagCapturedDialogFragment.java:1]: wait, no, that was an html issue, nevermind

Comment (wolfd) [reply app/src/main/java/website/bloop/app/dialogs/FlagCapturedDialogFragment.java:1]: Wow, github wtf.

Comment (wolfd) [reply app/src/main/java/website/bloop/app/dialogs/FlagCapturedDialogFragment.java:1]: wat-github

**Pull request merged**